src/Chapter1.hs

```diff
@@ -556,7 +570,8 @@ True
 >>> isVowel 'x'
 False
 -}
-isVowel c = error "isVowel: not implemented!"
+isVowel :: Char -> Bool
+isVowel = flip elem "AEIOUaeiou"
```

Comment:
Absolutely correct! And you use eta-reduction 👍🏼 

One note in here, that sometimes, `elem` could be slower than the explicit pattern matching. I remember there were some benchmarks on one particular case, that showed how moving to pattern matching on each case separately drastically decrease time 🐎

Reply:
Interesting, it makes sense since guards and case expressions just compile down to jump tables iirc.